Flaky test: GitRepositoryTest.testEmptyCommitWithRedundantRenames

```
GitRepositoryTest.testEmptyCommitWithRedundantRenames: FAILURE
com.linecorp.centraldogma.server.internal.storage.repository.git.GitRepositoryTest > testEmptyCommitWithRedundantRenames FAILED
    java.lang.AssertionError: 
    Expecting a throwable with cause being an instance of:
     <com.linecorp.centraldogma.server.internal.storage.repository.RedundantChangeException>
    but was an instance of:
     <com.linecorp.centraldogma.server.internal.storage.repository.ChangeConflictException: non-existent file/directory: Change{type=RENAME, path=/testEmptyCommitWithRedundantRenames/0.json}>
        at com.linecorp.centraldogma.server.internal.storage.repository.git.GitRepositoryTest.testEmptyCommitWithRedundantRenames(GitRepositoryTest.java:411)
```

--- a/server/src/test/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepositoryTest.java
+++ b/server/src/test/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepositoryTest.java
@@ -401,7 +401,7 @@ public class GitRepositoryTest {
     @Test
     public void testEmptyCommitWithRedundantRenames() throws Exception {
         // Create a file to produce redundant changes.
-        repo.commit(Revision.HEAD, Author.UNKNOWN, TEST_MESSAGE_SUMMARY, jsonUpserts[0]);
+        repo.commit(Revision.HEAD, Author.UNKNOWN, TEST_MESSAGE_SUMMARY, jsonUpserts[0]).join();
 
         // Ensure redundant changes do not count as a valid change.
         assertThatThrownBy(() -> repo.commit(Revision.HEAD, Author.UNKNOWN, TEST_MESSAGE_SUMMARY,

